ansible_base/authentication/social_auth.py

```
@@ -185,53 +199,76 @@ def ensure_strategy_in_args(self, args):
+        if email and social.email != email:
+            social.email = email
+            social.save(update_fields=["email"])
+        logger.info(
+            f"Stored OAuth email {email} for user {user.username} from {backend.database_instance.name}"
+        )
+    except AuthenticatorUser.DoesNotExist:
+        logger.warning(
+            f"No AuthenticatorUser found for uid={uid}, cannot store OAuth email for user {user.username}"
         )
-        logger.info(f"Stored OAuth email {email} for user {user.username} from {backend.database_instance.name}")
     except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)